docs/how-to/code/use-django-extension/task.yaml

```
@@ -0,0 +1,21 @@
+###########################################
+# IMPORTANT
+# Comments matter!
+# The docs use the wrapping comments as 
+# markers for including said instructions 
+# as snippets in the docs.
+###########################################
+summary: test the "Use the django-framework extension" guide
+
+environment:
+  ROCKCRAFT_ENABLE_EXPERIMENTAL_EXTENSIONS: "1"
+
+execute: |
+  cd example
+  rockcraft pack --verbosity debug
+  cd ../override_example
+  rockcraft pack --verbosity debug
+  cd ..
+
+restore: |
+  rm -rf */*.rock
```

```suggestion
environment:
  ROCKCRAFT_ENABLE_EXPERIMENTAL_EXTENSIONS: "1"
  EXAMPLE_DIR/example: example
  EXAMPLE_DIR/override_example: override_example

execute: |
  cd $EXAMPLE_DIR
  rockcraft pack --verbosity debug

restore: |
  cd $EXAMPLE_DIR
  rm *.rock
  rockcraft clean
```

This uses [variants](https://github.com/snapcore/spread?tab=readme-ov-file#variants) to break the test up. That way if `example` starts failing, it'll independently check whether `override_example` still succeeds.

---

Learned, thanks!